dockerfiles/state-signals-exporter-0.5.2-pod/state_signals_exporter.py

```diff
@@ -0,0 +1,60 @@
+
+import state_signals
+import time
+import argparse
+
+
+class StateSignalsExporter:
+    """
+    This class is run state signals exporter
+    """
+    def __init__(self, redis_host: str, scale_num: int, timeout: int):
+        self.__redis_host = redis_host
+        self.__scale_num = scale_num
+        self.__timeout = timeout
+
+    def state_signals_exporter(self):
+        """
+        This method run state_signals
+        :return:
+        """
+        sig_ex = state_signals.SignalExporter(redis_host=self.__redis_host, process_name="benchmark-runner", conn_timeout=self.__timeout)
+        print("\nBENCHMARK INIT TEST\n")
+        sig_ex.initialize_and_wait(
+            await_sub_count=self.__scale_num, legal_events=["benchmark-start"], periodic=True, timeout=self.__timeout
+        )
+        print(f"Proof of response (subs): {str(sig_ex.subs)}")
+        time.sleep(1)
+
+        print("\nBENCHMARK START TEST\n")
+        result, _ = sig_ex.publish_signal(
+            "benchmark-start", metadata={"something": "cool info"}, timeout=self.__timeout
+        )
+        print(f"SUBS CLEARED! Result code: {result}")
+        time.sleep(1)
+
+        print("\nBENCHMARK SHUTDOWN TEST\n")
```

Comment:
What's a test doing here?

Reply:
What do you mean ?

Reply:
What's test code doing in the production pod (I presume that's what "BENCHMARK SHUTDOWN TEST" means)?

Reply:
Its related to state-signals       
sig_ex.shutdown()

Reply:
Why the "TEST"?

Reply:
ok, will remove the word TEST

Reply:
OK